tests/interchain/provider_single_val_test.go

```diff
@@ -658,3 +618,127 @@ func (s *ProviderSuite) TestProviderOwnerChecks() {
 	s.Require().Equal(powerShapingParams.Top_N, uint32(consumerChain.PowerShapingParams.TopN))
 	s.Require().Equal(chainsuite.GovModuleAddress, consumerChain.OwnerAddress)
 }
+
+// Tests adding  and updating infraction parameters with MsgCreateConsumer, MsgUpdateConsumer
```

Comment:
Add the scenarios tested in the docstring.

Reply:
fixed [3a2f0e2](https://github.com/cosmos/interchain-security/pull/2416/commits/3a2f0e2864a2eaac768b0d8a63f45cb817e23b79)